include/Core/Posts/PostsModule.php

```diff
@@ -111,21 +125,170 @@ protected function do_delete( $options ) {
 	 * @return int Number of posts deleted.
 	 */
 	protected function delete_posts_from_query( $query, $options ) {
-		$query    = $this->build_query_options( $options, $query );
-		$post_ids = $this->query( $query );
+		$force_delete = isset( $options['force_delete'] ) ? $options['force_delete'] : false;
+		$query        = $this->build_query_options( $options, $query );
+		$post_ids     = $this->query( $query );
 
-		return $this->delete_posts_by_id( $post_ids, $options['force_delete'] );
+		return $this->delete_posts_by_id( $post_ids, $force_delete );
 	}
 
 	/**
 	 * Render the "private post" setting fields.
 	 */
 	protected function render_private_post_settings() {
-		if( $this->are_private_posts_present() ){
+		if ( $this->are_private_posts_present() ) {
 			bd_render_private_post_settings( $this->field_slug );
 		}
 	}
 
+	/**
```

Comment:
All these methods are moved to the `Renderer` method. Why are they added back to `PostsModules`?